### DISCUSSION TITLE
Not found: os

### DISCUSSION THREAD

**Discussion by abelardogit** [lihaoyi/os-lib#334] [General]
`
//> using scala 3.5.2
//> using dep "com.lihaoyi::os-lib:0.11.3"

@main
def hello(name: String): Unit =
  val paths = os.list(os.pwd)
`
from this [page](https://docs.scala-lang.org/getting-started/install-scala.html), throws this message:
![image](https://github.com/user-attachments/assets/be8f3c0e-063a-4cc9-8340-93c50ef950d7)

What's wrong, please? 
Thanks in advance. Brs.

**Comment by SethTisue** [comment]
@abelardogit are you using [scala-cli](https://scala-cli.virtuslab.org)? build tools such as sbt and mill don't know the `//> using` syntax. that syntax is specific to scala-cli. any other tool will just see it as a comment and ignore it
